cli/util/util.go

```diff
@@ -241,3 +242,12 @@ func GetLastNLines(filepath string, linesN int) ([]string, error) {
 
 	return lines, nil
 }
+
+// ReadEmbedFile reads content of embed file.
+func ReadEmbedFile(fs embed.FS, path string) (string, error) {
+	content, err := fs.ReadFile(path)
+	if err != nil {
+		return "", err
+	}
+	return string(content), nil
```

Comment:
Why convert to string, and if it is a binary file?

Reply:
Changed